source/horoscope/pages/landing.html

```diff
@@ -9,7 +9,21 @@
 </head>
 
 <body>
-    <h1 id="landing-title">Horoscope</h1>
+    <div class="header-container">
+        <h1 id="landing-title">Horoscope</h1>
+        <nav class="navbar">
+            <div class="menu-toggle" id="mobile-menu">
```

Comment:
Since we are custom creating the dropdown, its not keyboard interactable. Add `tabindex=0` to make the icon focusable by tab and then add a keypress listener to it so we can interact with it without a mouse. 

Other option is to use build in html `<select>` and `<option>` for drop down but you can style select but you can't style the drop down.

Reply:
also the `id='moble-menu"` confused me to think its something on mobile only but it isnt. Maybe `hamberger menu` might make more sense? or something else